test/Interop/Cxx/stdlib/use-std-span.swift

```diff
@@ -13,6 +14,24 @@ import CxxStdlib
 
 var StdSpanTestSuite = TestSuite("StdSpan")
 
+func takesSpanOfInt(s: Span) {
```

Comment:
Minor: we usually don't use argument labels for function parameters in these test utilities, unless they help with understanding of the semantics of the utility. In this case you could spell the function signature as `func takesSpanOfInt(_ s: Span)` to omit the argument label.

Reply:
Sure! I'll fix this